release/rllib_tests/learning_tests/yaml_files/cql/cql-halfcheetahbulletenv-v0.yaml

```diff
@@ -26,7 +26,7 @@ cql-halfcheetahbulletenv-v0:
         tau: 0.005
         target_entropy: auto
         no_done_at_end: false
-        n_step: 3
+        n_step: 1
```

Comment:
Wait, let's do this the other way around and keep the n_step setting.

So the new config logic is: `rollout_fragment_length` is "auto" b/c it's a lot of times dependent on other settings, like train_batch_size, num_envs_per_worker, and num_rollout_workers (PPO) or just needs to be at least `n_step` to make sense.

The RolloutWorkers now infer this setting from these other settings, if "auto", otherwise, take the user provided values, but this could lead to errors, like in this case.

So let's just remove the rollout_fragment_length setting entirely here (or set to its default: "auto").